### PR TITLE
users/config: consistency and readability tweaks

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

**Comment by rasa** [comment]
Re zeros vs zeroes, see https://english.stackexchange.com/questions/3824/what-is-the-plural-form-of-zero

File: users/config.rst [hunk 1]
```diff
@@ -338,61 +338,61 @@ order
...
+        Enabling this is highly discouraged - use at your own risk. You have been warned.
 
     When set to true, this device will pretend not to see instructions to
     delete files from other devices.
 
 scanProgressIntervalS
-    The interval with which scan progress information is sent to the GUI. Zero
-    means the default value (two seconds).
+    The interval in seconds with which scan progress information is sent to the GUI. Setting to ``0``
+    will force the system to use the default value of two.
```
Comment (tomasz1986) [inline users/config.rst:349]: This sounds rather lengthy. Is the "force the system" part really necessary? How about simply "will use the default value of two"?

Comment (rasa) [reply users/config.rst:349]: I agree, it sounds wordy, and "force" is heavy-handed. But just saying `0` will use `2` sounds non-sensical. It's best to spell things out in this case, as not everyone has English as their primary language.

I changed "force" to "cause". Is that better? If not, we'll simplify as you suggest.

Comment (tomasz1986) [reply users/config.rst:349]: > as not everyone has English as their primary language.

Personally, I'm not sure about the "system" part. Do you mean the operating system or Syncthing? Less technical users (regardless of their native tongue) will likely have no idea what "system" means here.

Comment (rasa) [reply users/config.rst:349]: Agreed. I changed it to Syncthing to be clear.

**Review by calmh — APPROVED** [review]

**Pull request merged**